2022-05-24 14:30:59,322 INFO    MainThread:85165 [wandb_setup.py:_flush():76] Loading settings from /Users/zed/.config/wandb/settings
2022-05-24 14:30:59,322 INFO    MainThread:85165 [wandb_setup.py:_flush():76] Loading settings from /Users/zed/workspace/VSCode/SJTU/nlp_proj/notebook/pl/wandb/settings
2022-05-24 14:30:59,322 INFO    MainThread:85165 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-05-24 14:30:59,322 INFO    MainThread:85165 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-05-24 14:30:59,322 INFO    MainThread:85165 [wandb_init.py:_log_setup():428] Logging user logs to /Users/zed/workspace/VSCode/SJTU/nlp_proj/notebook/pl/wandb/run-20220524_143059-2n0z9o8i/logs/debug.log
2022-05-24 14:30:59,322 INFO    MainThread:85165 [wandb_init.py:_log_setup():429] Logging internal logs to /Users/zed/workspace/VSCode/SJTU/nlp_proj/notebook/pl/wandb/run-20220524_143059-2n0z9o8i/logs/debug-internal.log
2022-05-24 14:30:59,323 INFO    MainThread:85165 [wandb_init.py:_jupyter_setup():378] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x16e7cc850>
2022-05-24 14:30:59,323 INFO    MainThread:85165 [wandb_init.py:init():462] calling init triggers
2022-05-24 14:30:59,323 INFO    MainThread:85165 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {}
2022-05-24 14:30:59,323 INFO    MainThread:85165 [wandb_init.py:init():515] starting backend
2022-05-24 14:30:59,323 INFO    MainThread:85165 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-05-24 14:30:59,329 INFO    MainThread:85165 [backend.py:ensure_launched():217] starting backend process...
2022-05-24 14:30:59,336 INFO    MainThread:85165 [backend.py:ensure_launched():222] started backend process with pid: 85289
2022-05-24 14:30:59,337 INFO    MainThread:85165 [wandb_init.py:init():525] backend started and connected
2022-05-24 14:30:59,351 INFO    MainThread:85165 [wandb_run.py:_label_probe_notebook():1089] probe notebook
2022-05-24 14:30:59,352 INFO    MainThread:85165 [wandb_run.py:_label_probe_notebook():1099] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-05-24 14:30:59,352 INFO    MainThread:85165 [wandb_init.py:init():596] updated telemetry
2022-05-24 14:30:59,354 INFO    MainThread:85165 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2022-05-24 14:31:01,863 INFO    MainThread:85165 [wandb_run.py:_on_init():1924] communicating current version
2022-05-24 14:31:04,869 INFO    MainThread:85165 [wandb_run.py:_on_init():1928] got version response 
2022-05-24 14:31:04,869 INFO    MainThread:85165 [wandb_init.py:init():662] starting run threads in backend
2022-05-24 14:31:09,880 INFO    MainThread:85165 [wandb_run.py:_console_start():1898] atexit reg
2022-05-24 14:31:09,880 INFO    MainThread:85165 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2022-05-24 14:31:09,881 INFO    MainThread:85165 [wandb_run.py:_redirect():1808] Wrapping output streams.
2022-05-24 14:31:09,881 INFO    MainThread:85165 [wandb_run.py:_redirect():1832] Redirects installed.
2022-05-24 14:31:09,881 INFO    MainThread:85165 [wandb_init.py:init():688] run started, returning control to user process
2022-05-24 14:31:09,882 INFO    MainThread:85165 [wandb_init.py:_pause_backend():341] pausing backend
2022-05-24 14:31:09,882 INFO    MainThread:85165 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-05-24 14:31:09,940 INFO    MainThread:85165 [wandb_init.py:_resume_backend():350] resuming backend
2022-05-24 14:31:09,942 INFO    MainThread:85165 [wandb_init.py:_pause_backend():341] pausing backend
2022-05-24 14:31:09,942 INFO    MainThread:85165 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-05-24 14:31:09,985 INFO    MainThread:85165 [wandb_init.py:_resume_backend():350] resuming backend
2022-05-24 14:31:46,125 INFO    MainThread:85165 [wandb_init.py:_pause_backend():341] pausing backend
2022-05-24 14:31:46,126 INFO    MainThread:85165 [jupyter.py:save_ipynb():377] not saving jupyter notebook
